### PR TITLE
Smart Contract Restaking Enhancements and Whitepaper Clarification

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

File: docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The following sections describe the steps to Restake "liquid" tokens (including
 1. Queue Withdrawal: invoke `DelegationManager.queueWithdrawal()` to trigger the escrow period. Wait for Escrow Period: 7 days. Please see further detail [here](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/#escrow-period-withdrawal-delay).
    * Parameters: please see the [QueuedWithdrawalParams](https://github.com/Layr-Labs/eigenlayer-contracts/blob/v0.3.2-mainnet-rewards/src/contracts/interfaces/IDelegationManager.sol#L93)
    * `strategy` - use the address of the deployed strategy ([example list here](https://github.com/Layr-Labs/eigenlayer-contracts?tab=readme-ov-file#deployments)).
+   * `shares` - the number of shares in the given strategy. Note this parameter is not meant to reference the amount of the underlying token. Invoke `[Strategy].underlyingToSharesView()` and `[Strategy].sharesToUnderlyingView()` as needed to convert their current balances between strategy shares and underlying token amounts.
```
Comment (wadealexc) [inline docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md:38]: You can use `underlyingToShares` and `sharesToUnderlying` -- the "View" tacked on to the end of both methods is not needed.

File: docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md [hunk 2]
```diff
@@ -215,12 +220,12 @@ Withdraw yield shares amount:
...
+## FAQ
 
 ### Queue withdrawal takes an `amount` as input, what will that value be?
 
 The input amount for `DelegationManager.queueWithdrawal()` can be any amount you like. However, it must be less than or equal to `withdrawableRestakedExecutionLayerGwei`. The value of `withdrawableRestakedExecutionLayerGwei` will equal any checkpointed yield (consensus rewards, fees, ETH) and any checkpointed exited validator native eth that has been withdrawn to the EigenPod.
 
 ### How to account for the exchange rates between Strategy token `amounts` and `shares`?
 
-Invoke `[Strategy].underlyingToSharesView()` and `[Strategy].sharesToUnderlyingView()` as needed to convert their current balances between shares and tokens.
+Invoke `[Strategy].underlyingToSharesView()` and `[Strategy].sharesToUnderlyingView()` as needed to convert their current balances between shares and underlying token amounts.
```
Comment (wadealexc) [inline docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md:231]: You can use `underlyingToShares` and `sharesToUnderlying` -- the "View" tacked on to the end of both methods is not needed.

Comment (wesfloyd) [reply docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md:231]: Awesome, thanks. Updated functions to remove View suffix.

File: docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md (hunk 1 above)
Comment (wesfloyd) [reply docs/eigenlayer/restaking-guides/1-restaking-developer-guide.md:38]: Awesome, thanks. Updated functions to remove View suffix.

**Comment by wesfloyd** [comment]
Thanks @wadealexc for the review!

@dabit3 if you can review/approve when convenient, we can merge live. Thank you

**Review by scotthconner — APPROVED** [review]

**Pull request merged**